modules/tracking/samples/tracker.py

```diff
@@ -2,38 +2,44 @@
 import cv2 as cv
 import sys
 
-if len(sys.argv) != 2:
-    print('Input video name is missing')
-    exit()
+#if len(sys.argv) != 2:
+#    print('Input video name is missing')
+#    exit()
```

Comment:
Is that necessary to change a sample?

Reply:
It was commented for more convenient experiments. For now it is not necessary to change it in final version.